snuba/clickhouse/http.py

```diff
@@ -325,4 +341,6 @@ def write(self, values: Iterable[bytes]) -> None:
             batch.append(value)
 
         batch.close()
-        batch.join()
+        # IMPORTANT: Please read the docstring of this method if you ever decide to remove the
+        # timeout argument from the join method.
+        batch.join(timeout=10)
```

Comment:
Can we make this configurable ? I feel we will eventually shoot ourselves in the feet without the ability to tweak it without merging a snuba change. 
It happened once with replays (the thread wait timeout on the parallel collector was hardcoded and broke during a weekend).

Reply:
Made it configurable with the default timeout of 10 seconds.